doc/source/troubleshooting.rst

```diff
@@ -0,0 +1,84 @@
+Troubleshooting
+===============
+
+.. note:: **Abstract**
+
+   This document describes situations which leads to issues
+   during build or boot time of the image build with {kiwi}.
+   The suggested solutions are considered best practice but
+   are just one out of other possible solution candidates.
+
+.. toctree::
+   :maxdepth: 1
+
+   troubleshooting/filesystems
+
+.. _supported-distributions:
+
+Build Host Constraints
+----------------------
+
+For building images a host system is required that runs the build process.
+Tools to create the image are used from that host and this creates an
+indirect dependency to the target image. For example; Building an
+Ubuntu image requires the apt and dpkg tools and metadata to be available
+and functional on the host to build an Ubuntu image. There are many more
+of those host vs. image dependencies and not all of them can be resolved
+in a clear and clean way.
+
+The most compatible environment is provided if the build host is of the same
+distribution than the target image. This always applies for the Open Build
```

Comment:
I was thinking that we could also mention the boxed plugin here, as this is an easy way to match host and target distro.

Reply:
got it and updated the docs

I think it's a good read now and easy to find